pkg/relay/ethereum/signer.go

```diff
@@ -71,7 +73,7 @@ func (s *EthereumSigner) Sign(address common.Address, tx *gethtypes.Transaction)
 		s.logger.Info("try to sign", "address", address, "txHash", txHash.Hex())
 	}
 
-	sig, err := s.bytesSigner.Sign(context.TODO(), txHash.Bytes())
+	sig, err := s.bytesSigner.Sign(s.ctx, txHash.Bytes())
```

Comment:
It's not good idea to save `ctx` in `Chain::Build()` and use it for every signing, right?
I think we should fix the design of `EthereumSigner` in the future, and for now, this argument should remain unchanged.

Reply:
Actually I also thought it was not good to add `ctx` field to `EhereumSigner`  and use it in this method, but I did so because it would be better to cancel the `Sign` method when the process receives a signal to stop.
However, I agree with you that we should fix the design in the future, so I have revered this change in https://github.com/datachainlab/ethereum-ibc-relay-chain/pull/64/commits/028bea00943143a18250cbf9ae51bfdb9973aae9